<?xml version="1.0"?>

<launch>

    
    <node name="hamster_acc_node" pkg="hamster_acc_pkg" type="hamster_acc_node" output="screen" />
    <node name="master_discovery_fkie" pkg="fkie_master_discovery" type="master_discovery" />
    <node name="master_sync_fkie" pkg="fkie_master_sync" type="master_sync" />


</launch>